### ISSUE TITLE
VS2017 Project references "util/libuv.py" which no longer exists

### ISSUE THREAD

**Issue by NoelFB** [munificent/wren#545]
The Visual Studio 2017 Project references a `util/libuv.py` script which no longer seems to exist in its Pre-Build command:
```
    <PreBuildEvent>
      <Command>python ../../libuv.py download
python ../../libuv.py build -32</Command>
    </PreBuildEvent>
```
I'm not sure how this should be updated but figured I'd mention it here!

**Comment by mathewmariani** [comment]
I sent a [pull request ](https://github.com/munificent/wren/pull/546) to address this issue.

**Comment by munificent** [comment]
Fix by #546 (I presume, I don't have a Windows machine to verify). Thanks, @mathewmariani!
